README.md

```diff
@@ -169,13 +169,15 @@ Table of contents:
 #### Server-Side Rendering
 * [Angular 2 Universal Repository (GitHub)](https://github.com/angular/universal)
 
-#### Material Design
+#### 
```

Comment:
Sorry if I caused problems. Do you mean I delete all the text I added and restore to the previous state or just arranging the lines 172 and 173 could the pull request be accepted? I would like to collaborate the best way as possible.